homeassistant/components/plugwise/config_flow.py

```diff
@@ -77,6 +84,31 @@ async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> Smile:
     return api
 
 
+async def verify_connection(
+    hass: HomeAssistant, user_input: dict[str, Any]
+) -> Smile | dict[str, str]:
+    """Verify and return the gateway connection or an error."""
+    errors: dict[str, str] = {}
+
+    try:
+        api = await validate_input(hass, user_input)
```

Comment:
```suggestion
        return await validate_input(hass, user_input)
```
that way we can remove the `else`